<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRInfoWidget</class>
 <widget class="QWidget" name="XRInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="moduleWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="moduleWidgetLabel">
        <property name="text">
         <string>Loaded modules:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="moduleListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="systemInfoWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="systemInfoWidgetLabel">
        <property name="text">
         <string>System info:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="systemInfoFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="systemInfoCpuCoresTitle">
           <property name="text">
            <string>CPU Cores:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="systemInfoCpuCoresValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="systemInfoCpuUsageLabel">
           <property name="text">
            <string>CPU Usage:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="systemInfoCpuUsageValue">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="systemInfoFreeMemoryTitle">
           <property name="text">
            <string>Free Memory:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="systemInfoFreeMemoryValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="systemInfoTotalMemoryTitle">
           <property name="text">
            <string>Total Memory:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="systemInfoTotalMemoryValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="systemInfoFreeSwapTitle">
           <property name="text">
            <string>Free Swap:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="systemInfoFreeSwapValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="systemInfoTotalSwapTitle">
           <property name="text">
            <string>Total Swap:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="systemInfoTotalSwapValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="systemInfoUsedMemoryLabel">
           <property name="text">
            <string>Memory Usage:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QProgressBar" name="systemInfoUsedMemoryValue">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="systemInfoUsedSwapLabel">
           <property name="text">
            <string>Swap Usage:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QProgressBar" name="systemInfoUsedSwapValue">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
